### ISSUE TITLE
nomad job scaling-events command does not work in 0.12.5.

### ISSUE THREAD

**Issue by CarelvanHeerden** [hashicorp/nomad#9318]
### Nomad version
```
 nomad --version
Nomad v0.12.5 (514b0d667b57068badb43795103fb7dd3a9fbea7)
```

### Operating system and Environment details
Ubuntu 18.04

### Issue
`nomad job scaling-events` command does not work in 0.12.5. In the release notes for 0.12.6 and 0.12.7, it does not mention that this was added in a later version. On the documentation, it does not mention the min version required: https://www.nomadproject.io/docs/commands/job/scaling-events

### Reproduction steps
```
nomad job scaling-events management
Usage: nomad job <subcommand> [options] [args]

  This command groups subcommands for interacting with jobs.

  Run a new job or update an existing job:

      $ nomad job run <path>

  Plan the run of a job to determine what changes would occur:

      $ nomad job plan <path>

  Stop a running job:

      $ nomad job stop <name>

  Examine the status of a running job:

      $ nomad job status <name>

  Please see the individual subcommand help for detailed usage information.
```

**Comment by tgross** [comment]
Hi @CarelvanHeerden! That command shipped in the 1.0.0-beta (see https://github.com/hashicorp/nomad/blob/master/CHANGELOG.md#100-unreleased). Unfortunately we don't yet have versioned docs on the website, but that's coming in the near future.

**Comment by CarelvanHeerden** [comment]
Thanks!
